things/api/http/requests.go

```diff
@@ -27,6 +28,10 @@ func (req addThingReq) validate() error {
 		return things.ErrUnauthorizedAccess
 	}
 
+	if len(req.Name) > maxNameSize {
+		return things.ErrMalformedEntity
+	}
+
```

Comment:
If we're going to introduce this limit, you should add it to DB schema too (use varchar(64)). Also, not sure if 64 is large enough. I would put at least 128. @drasko @nmarcetic @dusanb94 @srados what do you think?

Reply:
I agree! Especially with about size 64 not sounds to much.